roles/kubernetes_setup/tasks/main.yml

```diff
@@ -33,6 +33,39 @@
     - "{{ kube_controller_manager }}"
     - "{{ kube_scheduler }}"
 
+- name: Define kube 1.7 master nodes images
+  when: ansible_distribution == "Fedora" and ansible_distribution_major_version > '26'
+  set_fact:
+    kube_maj_ver: "1.7"
+    kube_apiserver: "registry.fedoraproject.org/f27/kubernetes-apiserver"
+    kube_controller_manager: "registry.fedoraproject.org/f27/kubernetes-controller-manager"
+    kube_scheduler: "registry.fedoraproject.org/f27/kubernetes-scheduler"
+
+- name: Define kube 1.7 worker nodes and etcd images
+  when: ansible_distribution == "Fedora" and ansible_distribution_major_version > '26'
+  set_fact:
+    kube_maj_ver: "1.7"
+    kube_kubelet: "registry.fedoraproject.org/f27/kubernetes-kubelet"
+    kube_proxy: "registry.fedoraproject.org/f27/kubernetes-proxy"
+    etcd: "registry.fedoraproject.org/f27/etcd"
+
```

Comment:
I thought of this and wrote it this way initially, but then realized the `kubernetes-node` pkg does not exist on f27AH and may never set the values for f27AH, hence changed it. 
Is there a reason we are checking specifically for `kubernetes-node` pkg, can I make it more generic like, just checking it for fedora and 26.

Reply:
Ah!, I know, we are also checking for Centos and RHEL I believe and hence so.

Reply:
Hmm...the exclusion of `kubernetes-node` on F27 might require additional rework to this role.  I'm not certain what happens if the RPM that is being checked via `rpm_version` is not there on the host.  

Maybe we need to add a `when:` to the use of the `rpm_version` so that it only runs on CentOS + RHEL?

@mike-nguyen any thoughts here?

Reply:
I just checked the `rpm_version` role and it will set the version to the standard output of `rpm -q --queryformat '%{VERSION}' rpm_name`.  When the package is not installed it returns `package rpm_name is not installed` which is not ideal.  

Maybe we should modify `rpm_version` with a when statement to only set_facty if 'is not installed' is not in standard output.  Then for the comparison, we can do the comparison AND `g_atomic_host['kubernetes-node'] is defined`

Reply:
Seems like a good approach; I would just rely on the return code rather than the stderr.  For example:

```
$ rpm -q --queryformat '%{VERSION}' foo
package foo is not installed
$ echo $?
1
$ rpm -q --queryformat '%{VERSION}\n' kernel
4.14.11
$ echo $?
0
```

As long as the RC == 0, we can save off the version.  @mike-nguyen can you make a PR for this change?